### ISSUE TITLE
Game setup proposition

### ISSUE THREAD

**Issue by RICCIARDI-Adrien** [opentomb/OpenTomb#524]
Hi,
I was thinking about a Qt program (or other cross platform technology) shipped close to OpenTomb binary that would act as Tomb Raider /setup configuration tool. It would allow to easily configure keys, graphics and sound (like the original one), but it could also use a simple drop box to select the Tomb Raider version to start. With this program, there is no need to implement such settings menus in game.
Program could automatically modify or create custom autoexec.lua and config.lua files, so unexperimented people could play the game easily.
What do you think about ?

**Comment by TeslaRus** [comment]
Good idea, but I think about SDL / GL based app with self written simple UI - good start for new code testing before adding settings UI to the main engine. Qt is good, but I do not want to add 10 MB of Qt dll's + runtime dll's only for layouts, edits and buttons.

**Comment by RICCIARDI-Adrien** [comment]
Yes, that's the point with Qt, it's dependencies are really heavy.
Furthermore, if you plan to add all settings to the main engine, there is no need to develop an additional program.
I was thinking of a way for the player to select the game he wants to start. If OpenTomb starts by displaying a main menu, it can propose a "new game" item, then a list of all available games (by searching for a specific file presence in data/trX directory for instance).
